_README.md

```diff
@@ -0,0 +1,145 @@
+<!--   
+The good readme should be easy to navigate through, therefore remember to add `markdown-toc` to devDependencies of your template and generate a table of contents by using the following script `"gen-readme-toc": "markdown-toc -i README.md"`
+-->
+
+<!-- toc -->
+
+- [Overview](#overview)
+- [Technical requirements](#technical-requirements)
+- [Specification requirements](#specification-requirements)
+- [Supported protocols](#supported-protocols)
+- [How to use the template](#how-to-use-the-template)
+  * [CLI](#cli)
+  * [Docker](#docker)
+- [Template configuration](#template-configuration)
+- [Custom hooks that you can disable](#custom-hooks-that-you-can-disable)
+- [Development](#development)
+- [Contributors](#contributors)
+
+<!-- tocstop -->
+
+## Overview
+
+<!--  
+The overview should explain in just a few sentences the template's purpose and its most essential features.
+-->
+
+This template generates an HTML page with an overview of the data model provided in your AsyncAPI file. Its core part is the diagram showing relations between the data model. Except for the HTML, you can also generate PDF from the HTML page or get the diagram into a separate PNG and SVG.
```

Comment:
```suggestion
This template generates an HTML page with an overview of the data model provided in your AsyncAPI file. Its core part is the diagram showing relations between the data model. Except for HTML, you can also generate PDFs from the HTML page or get the diagram into a separate PNG and SVG.

```

I would remove `the` in front of HTML :) and also perhaps `PDFs` sounds a bit better than `PDF` - if you want to keep it singular, use `a PDF (file)`